spec/events.spec.js

```diff
@@ -6,30 +6,141 @@ describe('events.spec: events namespace tests', () => {
 
     describe('events.on', () => {
         it('sets an event without throwing errors', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 function onTestEvent() {}
                 await Neutralino.events.on('testEvent', onTestEvent);
                 await __close('done');
             `);
             assert.equal(runner.getOutput(), 'done');
         });
+
+        it('triggers the event handler when an event is dispatched', async () => {
+            runner.run(`
+                let isEventTriggered = false;
+                function onTestEvent() {
+                    isEventTriggered = true;
+                    __close(JSON.stringify(isEventTriggered));
```

Comment:
Can we simply call `JSON.stringify({isEventTriggered: true})` without creating a new variable? Thanks :tada:

Reply:
Sure, I will make the changes